clients/unshaded/src/main/java/tachyon/client/file/FileSystem.java

```diff
@@ -0,0 +1,375 @@
+   */
+  void mount(TachyonURI src, TachyonURI dst, MountOptions options)
+      throws IOException, TachyonException;
+
+  /**
+   * Convenience method for {@link #openFile(TachyonURI, OpenFileOptions)} with default options.
+   *
+   * @param path the file to read from
+   * @return a {@link FileInStream} for the given path
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the given file does not exist
+   * @throws TachyonException if an unexpected Tachyon exception is thrown
+   */
+  FileInStream openFile(TachyonURI path)
+      throws FileDoesNotExistException, IOException, TachyonException;
+
+  /**
+   * Opens a file for reading.
+   *
+   * @param path the file to read from
+   * @param options options to associate with this operation
+   * @return a {@link FileInStream} for the given path
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the given file does not exist
+   * @throws TachyonException if an unexpected Tachyon exception is thrown
+   */
+  FileInStream openFile(TachyonURI path, OpenFileOptions options)
+      throws FileDoesNotExistException, IOException, TachyonException;
+
+  /**
+   * Convenience method for {@link #rename(TachyonURI, TachyonURI, RenameOptions)} with default
+   * options.
+   *
+   * @param src the path of the source, this must already exist
+   * @param dst the path of the destination, this path should not exist
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the given file does not exist
+   * @throws TachyonException if an unexpected Tachyon exception is thrown
+   */
+  void rename(TachyonURI src, TachyonURI dst)
+      throws FileDoesNotExistException, IOException, TachyonException;
+
+  /**
+   * Renames an existing Tachyon path to another Tachyon path in Tachyon. This operation will be
+   * reflected in the underlying storage if the path is persisted.
+   *
+   * @param src the path of the source, this must already exist
+   * @param dst the path of the destination, this path should not exist
+   * @param options options to associate with this operation
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the given file does not exist
+   * @throws TachyonException if an unexpected Tachyon exception is thrown
+   */
+  void rename(TachyonURI src, TachyonURI dst, RenameOptions options)
+      throws FileDoesNotExistException, IOException, TachyonException;
+
+  /**
+   * Convenience method for {@link #setAttribute(TachyonURI, SetAttributeOptions)} with default
+   * options.
+   *
+   * @param path the path to set attributes for
+   * @throws IOException if a non-Tachyon exception occurs
+   * @throws FileDoesNotExistException if the given file does not exist
+   * @throws TachyonException if an unexpected Tachyon exception is thrown
+   */
+  void setAttribute(TachyonURI path)
```

Comment:
If `setAttribute` is never called, would the attributes of the path be the same as default values in `SetAttributeOptions`? If yes, this convenience method doesn't need to exist since it doesn't change any attributes.